### PR TITLE
Package articles content

### PR DESCRIPTION
**Pull request by tmeasday**
Did a first pass at the writing packages article.

Will do the using packages article on this same branch (hopefully tomorrow).

Feel free to take a look or wait until it's all done :)

### CODE REVIEW

File: content/writing-packages.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+```js
+Package.onUse(function(api) {
+  api.mainModule('my-package-client.js', 'client');
+  api.mainModule('my-package-server.js', 'server');
+});
+```
+
+You can also add any source file that would be compiled to a JS file (such as a CoffeeScript file) in a similar way, assuming you [depend](#dependencies) on an appropriate build plugin.
+
+<h3 id="adding-css">Adding CSS</h3>
```
Comment (tmeasday) [inline content/writing-packages.md]: @benjamn says we should recommend `import`...

**Comment by tmeasday** [comment]
Ok, finished first pass at "Using Packages" too. Let me know what you think.

File: content/using-packages.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
+---
+title: Using Packages
+---
+
+After reading this article, you'll know:
+
+1. The third party packages systems you can use with Meteor, NPM and Atmosphere
```
Comment (stubailo) [inline content/using-packages.md]: This makes it sound like Meteor, NPM, and Atmosphere are package systems, maybe

"About NPM and Atmosphere, two JavaScript package systems you can use with Meteor"

File: content/using-packages.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+2. How to use NPM packages and deal with asyncronous APIs in synchronous contexts
```
Comment (stubailo) [inline content/using-packages.md]: "asyncronous APIs in synchronous contexts" -> "callbacks"

**Comment by stubailo** [comment]
@tmeasday we should move all of the content from here: http://guide.meteor.com/build-tool.html#using-packages

For example, how to override atmosphere packages locally

**Comment by stubailo** [comment]
The local vs. published section should also reference the above

File: content/writing-packages.md (hunk 1 above)
Comment (tmeasday) [inline content/writing-packages.md]: I'm not quite sure what he mean here, I can't make it work, will leave this for now, work on for 1.3.1

**Comment by tmeasday** [comment]
@stubailo I cleaned up the build tool article. This may be ready to merge now?

**Comment by stubailo** [comment]
Looks dope!

**Pull request merged**